src/main/java/com/uber/cadence/internal/worker/WorkflowWorker.java

```diff
@@ -250,23 +240,35 @@ private void sendReply(
 
   private class DecisionTaskWithHistoryIteratorImpl implements DecisionTaskWithHistoryIterator {
 
-    private long start = System.currentTimeMillis();
+    private final Duration RetryServiceOperationInitialInterval = Duration.ofMillis(200);
+    private final Duration RetryServiceOperationMaxInterval = Duration.ofSeconds(4);
+    private final Duration PaginationStart = Duration.ofMillis(System.currentTimeMillis());
+    private Duration DecisionTaskStartToCloseTimeout = Duration.ofSeconds(60);
```

Comment:
Where is this coming from? There is no default decision task timeout. It should come from the decision task.

Reply:
removed default value. This method will now throw if the response does not contain decision task scheduled